src/components/Section/Relationships/Relatives/Relative.jsx

```diff
@@ -844,6 +844,12 @@ export default class Relative extends ValidationElement {
 
             <Field
               title={i18n.t('relationships.relatives.heading.address.methods')}
+              comments={otherMethods}
+              commentsName="MethodsComments"
+              commentsValue={this.props.MethodsComments}
+              commentsActive={otherMethods}
+              commentsRequired={otherMethods}
```

Comment:
What's going on with the set of changes in this file? Was something not working before?

Reply:
@ryanhofdotgov yes, in the recent merge where a bunch of the "comments" fields were removed, these were removed when they shouldn't have been. These are both questions where if you answer "Other", you're asked to provide a comment.